### PR TITLE
COL-1736, we hope to reuse the old 'whiteboardToPng.js' script

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira-secure.berkeley.edu/browse/COL-1736

### CODE REVIEW

File: scripts/node_js/whiteboard_to_png.js [hunk 1]
```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/env node
+
+/**
+ * Copyright ©2021. The Regents of the University of California (Regents). All Rights Reserved.
```
Comment (lyttam) [inline scripts/node_js/whiteboard_to_png.js]: @johncrossman Should this be 2022?

Comment (johncrossman) [reply scripts/node_js/whiteboard_to_png.js]: Yes it should. Amending...

Comment (johncrossman) [reply scripts/node_js/whiteboard_to_png.js]: Amended. cc: @lyttam

Comment (lyttam) [reply scripts/node_js/whiteboard_to_png.js]: Thanks!

**Pull request merged**